Week3/homework/totalCost.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+{
+    function calculateTotalPrice (allOfItem) {
+        let price = 0;
+        for (let item in allOfItem) {
+            price = price + allOfItem[item];
+        }
+        return "The total price of all items " + price
```

Comment:
👍 but would like to introduce you to two new methods in JavaScript on objects and arrays i.e

- **Object.keys** https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
- **Array.prototype.reduce** - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce

Try to use both of these methods together and see if you can achieve the desired result using them.